### PR TITLE
delete accessPackageAssignmentRequest

### PR DESCRIPTION
**Pull request by markwahl-msft**
in entitlement management, had omitted the API to allow the admin to delete a denied or completed request.

### CODE REVIEW

File: api-reference/beta/api/accesspackageassignmentrequest-delete.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+# Delete accessPackageAssignmentRequest
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Delete an [accessPackageAssignmentRequest](../resources/accesspackageassignmentrequest.md) object.
+
+This request can be made to remove a denied or completed request.  You cannot delete an access package assignment requset if it has any **accessPackageAssignment**.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignmentrequest-delete.md:18]:
```suggestion
This request can be made to remove a denied or completed request. You cannot delete an access package assignment request if it has any **accessPackageAssignment** objects.
```
```suggestion
This request can be made to remove a denied or completed request.  You cannot delete an access package assignment requset if it has any **accessPackageAssignment**.
```

File: api-reference/beta/api/accesspackageassignmentrequest-delete.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### Request
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "delete_accesspackageassignmentrequest"
+}-->
+
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignmentrequest-delete.md:62]: Missing a request object here?

File: changelog/Microsoft.DirectoryServices.json [hunk 3]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+       {
+         "Id": "aa34ca74-c3b4-4981-ba1c-809256deacb0",
+         "ApiChange": "Method",
+         "ChangedApiName": "Delete",
+         "ChangeType": "Addition",
+         "Description": "Added the [delete](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentrequest-delete?view=graph-rest-beta) operation for an [accessPackageAssignmentRequest](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentrequest?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:10]:
```suggestion
         "Description": "Added the [delete](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentrequest-delete?view=graph-rest-beta) operation to the [accessPackageAssignmentRequest](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentrequest?view=graph-rest-beta) resource.",
```
```suggestion
         "Description": "Added the [delete](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentrequest-delete?view=graph-rest-beta) operation for an [accessPackageAssignmentRequest](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentrequest?view=graph-rest-beta) resource.",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
- Proposed minor fixes
- You'll need to update the localization metadata for all files from _localization_priority: Normal_ to **ms.localizationpriority: medium**. Otherwise, you'll hit an API Doctor validation error.

File: api-reference/beta/api/accesspackageassignmentrequest-delete.md (hunk 1 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackageassignmentrequest-delete.md]: fixed typo, thanks

File: api-reference/beta/api/accesspackageassignmentrequest-delete.md (hunk 2 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackageassignmentrequest-delete.md]: added, thanks

File: changelog/Microsoft.DirectoryServices.json (hunk 3 above)
Comment (markwahl-msft) [reply changelog/Microsoft.DirectoryServices.json]: updated

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**